README.md

```diff
@@ -437,7 +437,7 @@ onSet(store, ({ newValue, abort }) => {
 })
 ```
 
-Event listeners can communicate with `payload.shared` object.
+Same event listeners can communicate with `payload.shared` object.
```

Comment:
To be honest, I expected the `shared` object to be available across different lifecycle methods.
Because of such confusion, I added this kind of info to the README.

Reply:
Is it hard to add `shared` for everyone? Will it affect performance?

Reply:
> Is it hard to add `shared` for everyone? Will it affect performance?

Idk for now, I need time to deal with the codebase.
However, it seems like a topic for a separate PR

Reply:
> However, it seems like a topic for a separate PR

Totally agree